orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/purple/PurpleGeneCopyNumber.java

```diff
@@ -13,6 +13,11 @@ public interface PurpleGeneCopyNumber
     @NotNull
     String gene();
 
+    @NotNull
+    String transcript();
+
+    boolean isCanonical();
+
```

Comment:
In `LinxBreakend` these fields appear after chromosome. Would suggest to make it consistent